src/components/POLine/POLineForm.js

```diff
@@ -62,7 +62,8 @@ class POLineForm extends Component {
     deletePOLine: PropTypes.func,
     change: PropTypes.func,
     dispatch: PropTypes.func,
-  }
+    vendor: PropTypes.object.isRequired,
```

Comment:
basically it's a valid case for now - vendor is deleted and it can't be retrieved from back-end. I think we have to support such case and display form with po line

Reply:
Fixed, but from me it's a bit weird case. Vendor is required field for PO and allow deleting vendor which is used in PO it's wrong approach